pino-eventhub.js

```diff
@@ -35,24 +36,30 @@ function pinoEventHub (opts) {
     return line
   })
 
-  var url = decodeURIComponent(opts.sr) + '/messages'
-  var options = {
+  const url = decodeURIComponent(opts.sr) + '/messages'
+  const options = {
```

Comment:
we should also have an http.Agent created, with a configurable number of sockets attached.